src/GitLinkTask/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "NuProj.Common": "0.11.14-beta"
```

Comment:
I don't know if it works. I know NuProj's build defines custom MSBuild tasks, but I _expect_ XBuild supports that.
Is building on linux important (as opposed to just running on linux)?